### PR TITLE
Issue 153

### PR DESCRIPTION
**Pull request by nicolevasilevsky**

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
@monicacecilia Shahim got me all sorted out, and I think all the tasks have been addressed in #153 in this PR.

File: public/team.yaml [hunk 1]
```diff
@@ -185,7 +185,17 @@ institutions:
         title: Scientific Curator
         picture: img/team/CarmodyLeigh_headshot_Nov2016.jpg
         alumni: false
+        bio:  Michael is a software engineer, working to make a difference where biology and computer science meet. He specializes in full-stack web applications, scripting, and bioinformatics related technologies such as bioPerl, bioPython, and NGS frameworks. He has additional interests in machine learning and data mining algorithms. He aims to bridge the gap between the vast amounts of data and the need for results or visualizations.
```
Comment (monicacecilia) [inline public/team.yaml]: Oops, an accidentally misplaced insertion here. :bowtie: Leigh has Mike's bio and bio has Leigh's bio. Just swap those two.

Comment (nicolevasilevsky) [reply public/team.yaml]: fixed!

Comment (monicacecilia) [reply public/team.yaml]: Brilliant! Thanks!

**Comment by monicacecilia** [comment]
Fixes #153

**Pull request merged**